pinot-core/src/test/java/org/apache/pinot/queries/BaseQueriesTest.java

```diff
@@ -264,10 +271,22 @@ private BrokerResponseNative getBrokerResponseDistinctInstances(PinotQuery pinot
     Plan plan1 = planMaker.makeInstancePlan(instances.get(0), serverQueryContext, EXECUTOR_SERVICE, null);
     Plan plan2 = planMaker.makeInstancePlan(instances.get(1), serverQueryContext, EXECUTOR_SERVICE, null);
 
-    DataTable instanceResponse1 =
-        queryContext.isExplain() ? ServerQueryExecutorV1Impl.processExplainPlanQueries(plan1) : plan1.execute();
-    DataTable instanceResponse2 =
-        queryContext.isExplain() ? ServerQueryExecutorV1Impl.processExplainPlanQueries(plan2) : plan2.execute();
+    DataTable instanceResponse1;
+    try {
+      instanceResponse1 =
+          queryContext.isExplain() ? ServerQueryExecutorV1Impl.processExplainPlanQueries(plan1) : plan1.execute();
+    } catch (TimeoutException e) {
+      e.printStackTrace();
```

Comment:
We don't typically do `e.printStackTrace();`

Reply:
fixed